base/applications/rapps/loaddlg.cpp

```diff
@@ -982,12 +982,31 @@ CDownloadManager::ThreadFunc(LPVOID param)
         // run it
         if (InfoArray[iAppId].DLType == DLTYPE_APPLICATION)
         {
-            SHELLEXECUTEINFOW shExInfo = {0};
-            shExInfo.cbSize = sizeof(shExInfo);
+            // FIXME: Unable to open ZIP/7Z/RAR files by ShellExecute.
+            //        Sorry but I use a workaround by opening parent folder.
+            static const LPCWSTR s_CanOpenDotExtsCurrently[] =
+            {
+                L".exe", L".com", L".bat", L".reg", L".msi"
```

Comment:
```suggestion
                L".exe",
                L".com",
                L".bat",
                L".reg",
                L".msi"
```
Might improve "git blame" if we would ever need to extend the list.